Installer/package.txt

```diff
@@ -450,6 +450,23 @@ component Extensions
 	into [[ToolsFolder]]/Extensions place
 		rfolder macosx:packaged_extensions/com.livecode.library.scriptitems
 
+    into [[ToolsFolder]]/Extensions place
```

Comment:
I think the indenting is wrong here. These files are _very_ fussy about indenting